backend/src/LocalExec/Libs/List.fs

```diff
@@ -0,0 +1,71 @@
+module LocalExec.Libs.List
+
+open Prelude
+open LibExecution.RuntimeTypes
+open LibExecution.StdLib.Shortcuts
+
+
+module Errors = LibExecution.Errors
+module Interpreter = LibExecution.Interpreter
+
+let varA = TVariable "a"
+
+let types : List<BuiltInType> = []
+let constants : List<BuiltInConstant> = []
+
+
+let fns : List<BuiltInFn> =
+  [ { name = fn [ "LocalExec"; "StdLib"; "List" ] "iter" 0
```

Comment:
Maybe a different word from StdLib. Perhaps Builtins or a similar synonym, to indicate why we're using these.